pkg/datapath/iptables/iptables.go

```diff
@@ -940,6 +968,19 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 	}
 
 	for _, c := range ciliumChains {
+		// do not install feeder for chains that are set to be disabled
+		skipFeeder := false
+		for _, disabledChain := range option.Config.DisableIptablesFeederRules {
+			if strings.ToLower(c.hook) == strings.ToLower(disabledChain) {
+				log.Infof("Skipping the install of feeder for chain %s", c.hook)
```

Comment:
nit: To help parsing logs in log management systems, we prefer the form more like this:

```suggestion
				log.WithField("chain", c.hook).Info("Skipping the install of feeder rule")
```